openshift/backend.bc.yaml

```diff
@@ -2,19 +2,16 @@
 apiVersion: v1
 kind: Template
 labels:
+  build: "${REPO_NAME}-backend"
```

Comment:
Just a heads up - there are name length limitations in OpenShift.  24 characters I believe.  Using repo name is not a big deal for nr-get-token, but other projects (ex. nr-messaging-service-showcase) takes up a lot of the available characters. So, probably best not to used repo name in any names or labels... (for next project)

Reply:
👍